resources/mon.rb

```diff
@@ -43,10 +43,17 @@
     creates "/var/lib/ceph/mon/ceph-#{node['hostname']}/done"
   end
 
+  # Docker has extra protections in place which are preventing the execution of the next resource
+  execute 'Disable fs issue' do
+    command 'sysctl -w fs.protected_regular=0'
+    action :nothing
+  end
+
```

Comment:
Please use the [sysctl resource](https://docs.chef.io/resources/sysctl/) instead.